tests/Dockerfile

```diff
@@ -18,22 +18,9 @@ RUN env-build-tool new /conf/requirements.txt ${py_env_path} /wheels
 # Install the tools
 ADD ./ /code
 WORKDIR /code
-# TODO: work out why this needs `--editable`
-RUN /env/bin/pip install --extra-index-url https://packages.dea.ga.gov.au/ \
-    --editable apps/dc_tools[tests] \
-    --editable apps/cloud \
-    --editable apps/dnsup \
-    --editable libs/algo \
-    --editable libs/aio \
-    --editable libs/aws \
-    --editable libs/dscache \
-    --editable libs/geom \
-    --editable libs/index \
-    --editable libs/io \
-    --editable libs/ppt \
-    --editable libs/stats \
-    --editable libs/thredds \
-    --editable libs/ui
+
+# This installs things twice because of circular dependencies
```

Comment:
there no circular dependencies, the reason for doing it twice is to ensure installation installs dev versions of code

If installing just two libraries A and B with "A needs B" dependency, pip will install A (and B from pypi, because A needs it), then it won't install local B, because there is B installed already.

A->B
B